### DISCUSSION TITLE
View commit history of individual files

### DISCUSSION THREAD

**Discussion by jmarshall** [community/community#5706] [Mobile] [answered]
When you drill down to look at a directory or file on the repository web view, there is a “History” button that takes you to the list of commits affecting that particular directory/file:

<img width="811" alt="Screenshot 2021-09-18 at 10 57 22" src="https://user-images.githubusercontent.com/70921/133885000-b222086f-d3dc-40ac-bc56-bacf2d1c5f95.png">

It would be good if there were a way to do this on the mobile app too. When you drill down to look at the directory tree with “Browse code”, for example the ••• menu at top-right could additionally contain a “History” menu item that would take you to the list of commits affecting the particular directory or file that you are viewing.

**Comment by jmarshall** [comment, marked as answer]
It appears this has now been implemented, e.g., in version 1.94 of the iOS app.

**Comment by haveAnIssue** [comment]
On iPhone website I don’t see “History”.
